### PR TITLE
[FIX] stock_account: Don't fail on period not found in inventory_period_id

### PR DESCRIPTION
**Pull request by pedrobaeza**
You can make inventories without having a "Force account period" set, and even you can use inventory without account and thus not having created accounting periods.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: addons/stock_account/migrations/8.0.1.1/post-migrate.py [hunk 1]
```diff
@@ -91,10 +91,15 @@ def inventory_period_id(cr, pool):
...
-            cr, SUPERUSER_ID, dt=date[:10], context={'company_id': company_id})
-        inventory_obj.write(
-            cr, SUPERUSER_ID, inv_id, {'period_id': period_ids[0]})
+        try:
+            period_ids = period_obj.find(
+                cr, SUPERUSER_ID, dt=date[:10],
+                context={'company_id': company_id})
+            inventory_obj.write(
+                cr, SUPERUSER_ID, inv_id, {'period_id': period_ids[0]})
+        except:
```
Comment (hbrunn) [inline addons/stock_account/migrations/8.0.1.1/post-migrate.py]: better catch only `openerp.exceptions.RedirectWarning`

**Comment by StefanRijnhart** [comment]
@pedrobaeza can you catch the specific exception that Holger mentioned? Then it's :+1: for me.

**Comment by pedrobaeza** [comment]
Done

**Comment by StefanRijnhart** [comment]
Thanks! Merging because Holger's comment was honoured.

**Pull request merged**